modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/Layout/LayoutScreen/ModalAddObjectLayoutTab.tsx

```diff
@@ -270,7 +270,7 @@ export function ModalAddObjectLayoutTab({
 								});
 							}}
 							required
-							selectedKey={selectedRelationship?.name}
+							selectedKey={selectedRelationship?.id.toString()}
```

Comment:
Using the same approach from [here](https://github.com/carolmariaabb/liferay-portal/blob/38131bd1c68de68ae139a595824a7acdf2191b2c/modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModalAddFilter.tsx#L454-L481).